DOKS-wordpress/README.md

```diff
@@ -0,0 +1,428 @@
+## Overview
+
+[WordPress](https://wordpress.org/about/) is an Open Source software designed for everyone, emphasizing accessibility, performance, security, and ease of use to create a website, blog, or app. [WordPress](https://en.wikipedia.org/wiki/WordPress) is a content managment system (CMS) built on PHP and using MySQL as a data store, powering over 30% of internet sites today.
+
+In this tutorial, we’ll use Helm for setting up [WordPress](https://wordpress.com/) on top of a Kubernetes cluster, in order to create a highly-available website. In addition to leveraging the intrinsic scalability and high availability aspects of Kubernetes, this setup will help keeping WordPress secure by providing simplified upgrade and rollback workflows via Helm.
```

Comment:
Please use "you/you will" instead of "we/we'll". Same applies for every step in the tutorial.